app/models/create_draft_edition.rb

```diff
@@ -39,6 +39,10 @@ def document
   def fill_out_new_edition
     document.increment!(:stale_lock_version)
     set_first_published_at
+    set_publisher_first_published_at
+    set_publisher_major_published_at
+    set_publisher_minor_published_at
+    set_publisher_last_edited_at
     set_last_edited_at
     set_document_owner
```

Comment:
I think we want the `temporary_{major,minor}_published_at` date fields here to be set using the value in the `previously_published_item`. In this way, an edition with a major `update_type` still has the minor date set for the last time that document had a minor update.